qcodes/utils/slack.py

```diff
@@ -0,0 +1,418 @@
+import os
+import tempfile
+from functools import partial
+from time import sleep
+import inspect
+from slacker import Slacker
+import threading
+import traceback
+
+from qcodes.plots.base import BasePlot
+from qcodes import config as qc_config
+from qcodes.instrument.parameter import _BaseParameter
+from qcodes import active_loop, active_data_set
+
+
+def convert_command(text):
```

Comment:
Not sure what you mean, the try_convert_str is a function that's only called within convert_command, so I made it a subfunction.